docs/quickstart.rst

```diff
@@ -3,24 +3,93 @@ Quick Start
 For now, there isn't much you can do with the Garden service, but this is what
 currently works.
 
-Install Garden Client
----------------------
+Install Garden library and CLI
+------------------------------
 The garden client has been published to pypi, so the following will install the
 client in your current python envioronment::
 
-    pip install garden-ai
+    $ pip install garden-ai
+
+If you have it, we recommend installing the CLI with `pipx`_.::
+
+    $ pipx install garden-ai
+
+You should then be able to see the following: ::
+
+    $ garden create --help
+
+    Usage: garden create [OPTIONS] [DIRECTORY]
+
+    Create a new Garden
+
+    ╭─ Arguments ───────────────────────────────────────────────────────────────────╮
+    │   directory      [DIRECTORY]  (Optional) if specified, this generates a       │
+    │                               directory with subfolders to help organize the  │
+    │                               new Garden. This is likely to be useful if you  │
+    │                               want to track your Garden/Pipeline development  │
+    │                               with GitHub.                                    │
```

Comment:
Noting for follow-up work: Let's revisit this when we get to scaffolding pipelines. This help text suggests that invoking this directory option on garden creation is _the_ way to track your project in GitHub. This contradicts what I think will be our standard/encouraged path: maintaining your pipeline code in your pre-existing repo.

Reply:
Ah I was trying to be as noncommittal as I could with the help text -- probably worth rephrasing before we demo but agree let's leave that as follow-up